layouts/default.vue

```diff
@@ -2,15 +2,15 @@
   <ColorScheme tag="span">
     <div class="relative mx-auto flex min-h-screen max-w-7xl flex-col">
       <header
-        class="dark:border-tertiary sticky left-0 top-0 z-50 flex h-14 items-center bg-light bg-opacity-95 p-2 dark:bg-dark dark:bg-opacity-95"
+        class="dark:border-tertiary sticky left-0 top-0 z-50 flex h-16 items-center bg-light bg-opacity-95 p-4 dark:bg-dark dark:bg-opacity-95"
       >
         <TheNavigation />
       </header>
-      <main class="h-0 flex-grow overflow-y-auto px-2 py-9 md:px-4">
```

Comment:
War das Problem die fixe height vom main content und das overflow-Y-auto?

Konntest du das genau nachvollziehen, warum das Problem aufgetreten ist?

Reply:
I guess I just overdid it with the layout. A bit over engineering